src/lib/openapi/spec/bulk-metrics-schema.ts

```diff
@@ -11,12 +11,15 @@ export const bulkMetricsSchema = {
         'A batch of metrics accumulated by Edge (or other compatible applications). Includes both application registrations as well usage metrics from clients',
     properties: {
         applications: {
+            description: 'a list of applications registered by Unleash SDK',
```

Comment:
I think we're missing an article in front of "unleash sdk". Is it "an"? "the"? It's hard to tell. Or is it plural? 
```suggestion
            description: 'A list of applications registered by (an/the)? Unleash SDK(s?)',
```